src/armorblox/api/base_api.py

```diff
@@ -51,10 +51,12 @@ def list_resource(self, path: str, headers: dict = None,
         p = self.list_params()
         if params is not None:
             p.update(params)
+            # Cap page size to 50 for optimal performance
```

Comment:
```suggestion
            # Cap page size to PAGE_SIZE value for optimal performance
```
I added this change as an edit earlier, but it had got merged by then.